modules/codemod/lib/v5/spec/removeButtonEnums.spec.ts

```diff
@@ -0,0 +1,390 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transform from '..';
+
+const expectTransform = expectTransformFactory(transform);
+
+describe('removeButtonEnum', () => {
+  it('should remove variant, size, and position enum imports since they no longer exist', () => {
+    const input = `
+      import {
+        Button,
+        ButtonVariant,
+        IconButtonVariant,
+        OutlineButtonVariant,
+        DropdownButtonVariant,
+        TextButtonVariant,
+        DeprecatedButtonVariant,
+        ButtonIconPosition,
+        ButtonSize,
+      } from '@workday/canvas-kit-react/button';
+    `;
+
+    const expected = `
+      import { Button } from '@workday/canvas-kit-react/button';
+    `;
+
+    expectTransform(input, expected);
+  });
```

Comment:
This way of writing codemod tests aligns more with all our other tests of having well-defined `describe` and `it` blocks. The `input` and `expected` constants have enough of a visual separation to more easily understand via syntax highlighting where the input string and expected strings start and end. The `it` block also allows for `.only` for test isolation.